geometry_msgs/Pose gripper_pose # Pose of gripper with respect to object during grasp
float64 gripper_width           # Width of gripper when object is grasped
